src/CSF.SRDashboard.Client/Components/Tables/Dashboard/DashboardTable.razor.cs

```diff
@@ -34,6 +38,16 @@ public void RowClicked(Document document)
         {
             this.NavigationManager.NavigateTo($"SeafarerProfile/{this.Applicant.Cdn}/view-attachment/{document.DocumentId}");
         }
-
+        private string formatDocumentTypesAsString(Document document)
+        {
+            string docTypeString = string.Empty;
+            foreach(var docTypes in document.DocumentTypes)
+            {
+                docTypeString += docTypes.Description + ", ";
+            }
+            docTypeString = docTypeString.Remove(docTypeString.Length - 2);
+            
+            return docTypeString;
```

Comment:
This whole statement can be replaced with this oneliner. Does exactly the same thing.
```suggestion
            return string.Join(", ", document.DocumentTypes.Select(x => x.Description).ToArray());
```